native-image-tests/build.gradle

```diff
@@ -16,12 +30,14 @@ animalsniffer {
 }
 
 def graalHome = "/Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.2.0/Contents/Home"
-def graalvmResources = project.file("$buildDir/graalvm/resources")
-def graalvmConfig = project.file("$graalvmResources/META-INF/native-image")
+def graalvmResources = project.file("src/main/resources")
+def graalvmConfig = project.file("$graalvmResources/META-INF/native-image/native-image-tests")
 graalvmConfig.mkdirs()
 
 sourceSets {
   main.resources.srcDirs += graalvmResources
+
+  main.java.srcDirs += project.file("../okhttp-sse/src/test/java")
```

Comment:
Idea 4: *gradle symlink* the existing classes in, starting with a small fast module but building up.